examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,279 @@
+    category = CuriePME.classify(vector, vectorNumBytes);
+
+    if (category == CuriePME.noMatch) {
+        Serial.println("Don't recognise that one-- try again.");
+    } else {
+        letter = category + upperStart;
+        Serial.println(letter);
+    }
+}
+
+/* Simple "moving average" filter, removes low noise and other small
+ * anomalies, with the effect of smoothing out the data stream. */
+byte getAverageSample(byte samples[], unsigned int num, unsigned int pos,
+                   unsigned int step)
+{
+    unsigned int ret;
+    unsigned int size = step * 2;
+
+    if (pos < (step * 3) || pos > (num * 3) - (step * 3)) {
+        ret = samples[pos];
+    } else {
+        ret = 0;
+        pos -= (step * 3);
+        for (unsigned int i = 0; i < size; ++i) {
+            ret += samples[pos - (3 * i)];
+        }
+
+        ret /= size;
+    }
+
+    return (byte)ret;
+}
+
+/* We need to compress the stream of raw accelerometer data into 128 bytes, so
+ * it will fit into a neuron, while preserving as much of the original pattern
+ * as possible. Assuming there will typically be 1-2 seconds worth of
+ * accelerometer data at 200Hz, we will need to throw away over 90% of it to
+ * meet that goal!
+ *
+ * This is done in 2 ways:
+ *
+ * 1. Each sample consists of 3 signed 16-bit values (one each for X, Y and Z).
+ *    Map each 16 bit value to a range of 0-255 and pack it into a byte,
+ *    cutting sample size in half.
+ *
+ * 2. Undersample. If we are sampling at 200Hz and the button is held for 1.2
+ *    seconds, then we'll have ~240 samples. Since we know now that each
+ *    sample, once compressed, will occupy 3 of our neuron's 128 bytes
+ *    (see #1), then we know we can only fit 42 of those 240 samples into a
+ *    single neuron (128 / 3 = 42.666). So if we take (for example) every 5th
+ *    sample until we have 42, then we should cover most of the sample window
+ *    and have some semblance of the original pattern. */
+void undersample(byte samples[], int numSamples, byte vector[])
+{
+    unsigned int vi = 0;
+    unsigned int si = 0;
+    unsigned int step = numSamples / samplesPerVector;
+    unsigned int remainder = numSamples - (step * samplesPerVector);
+
+    /* Centre sample window */
+    samples += (remainder / 2) * 3;
+    for (unsigned int i = 0; i < samplesPerVector; ++i) {
+        for (unsigned int j = 0; j < 3; ++j) {
+            vector[vi + j] = getAverageSample(samples, numSamples, si + j, step);
+        }
+
+        si += (step * 3);
+        vi += 3;
+    }
+}
+
+void readVectorFromIMU(byte vector[])
+{
+    byte accel[sensorBufSize];
+    int raw[3];
+
+    unsigned int samples = 0;
+    unsigned int i = 0;
+
+    /* Wait until Motion detection inturrupt flag  */
+    while (motionFlag == false)
+{
+    digitalWrite(buzzerPin, LOW);  // Optional Buzzer off
+}
+    /* While Motion detection inturrupt flag being held... */
+    while (motionFlag == true) {
+
+        if (CurieIMU.accelDataReady()) {
```

Comment:
I'll repeat myself, again, accelDataReady does not exist anymore. Please look at the latest DrawingInTheAir example.
And update your version of corelibs-arduino101 from github, because this should not be compiling for you.